### ISSUE TITLE
Could you please upload mlmodels somewhere else? or without Github LFS ?

### ISSUE THREAD

**Issue by XBeg9** [mdering/CoreMLZoo#2]
I am getting this when cloning repository:

`Downloading SqueezeNet_v1.0.mlmodel (4.77 MB)
Error downloading object: SqueezeNet_v1.0.mlmodel (927328c): Smudge error: Error downloading SqueezeNet_v1.0.mlmodel (927328ccc281bb62491143b689246785662de353a7ca50926e08178a57854a13): batch response: This repository is over its data quota. Purchase more data packs to restore access.`

Thanks in advance!

**Comment by mdering** [comment]
Yes I got an email about this. Any suggestions for alternatives?

**Comment by toXel** [comment]
@mdering Maybe you can upload the SqueezeNet models directly with git? ~ 5 MB is not that huge so you don't need LFS for it.

**Comment by rubenvde** [comment]
Any update about this? Is there a download link to download the VGG19.mlmodel? I can try and convert it with the coremltools but if you already got the file is there a possibility to get it?
